Content.Client/CombatMode/ShowCombatModeIndicatorsOverlay.cs

```diff
@@ -0,0 +1,113 @@
+using Content.Shared.CCVar;
+using Content.Client.Hands.Systems;
+using Content.Shared.Weapons.Ranged.Components;
+using Robust.Client.GameObjects;
+using Robust.Client.Graphics;
+using Robust.Client.Input;
+using Robust.Client.UserInterface;
+using Robust.Client.Player;
+using Robust.Shared.Configuration;
+using Robust.Shared.Enums;
+using Robust.Shared.Utility;
+
+namespace Content.Client.CombatMode
+{
+    public sealed class ShowCombatModeIndicatorsOverlay : Overlay
+    {
```

Comment:
File-scoped namespaces and commenting.

Reply:
please expand this I don't understand this short note

Reply:
use namespace Content.Client.CombatMode; and add a doc comment for what the overlay is

Reply:
i have already used namespace Content.Client.CombatMode. Or did I use something wrong?

Reply:
i added a summary comment for class